tests/hyperv_test.go

```diff
@@ -0,0 +1,259 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2019 Red Hat, Inc.
+ *
+ */
+
+package tests_test
+
+import (
+	"flag"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"fmt"
+
+	tframework "kubevirt.io/kubevirt-ansible/tests/framework"
+	"kubevirt.io/kubevirt/pkg/kubecli"
+	ktests "kubevirt.io/kubevirt/tests"
+	"time"
+	k8sv1 "k8s.io/api/core/v1"
+//	"os"
+//	"regexp"
+//	"strings"
+//	"sync"
+    	"io/ioutil"
+	"github.com/mholt/archiver"
+	"reflect"
+	"encoding/json"
+	"encoding/xml"
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
+)
+
+const (
+	windowsDisk         = "windows-disk"
+	username            = "Administrator"
+	password            = "Heslo123"
```

Comment:
I wonder if there is a better way than hardcode passwords in the source code

Reply:
Agree, but same way uses in the unit tests. Probably it's not important image. For new test framework we can write some kind of *infrastructure to store secrets*